### ISSUE TITLE
`reductions(acc, value)` vs `fold(value, acc)`

### ISSUE THREAD

**Issue by Raynos** [Gozala/reducers#43]
It's confusing to have both signatures.

I made the mistake of doing `reductions(function (v, acc) { ... })`

**Comment by Gozala** [comment]
You're right, although I would still prefer `fold(input, function(acc, v) { ... })` since every other language follows that pattern.

**Comment by Raynos** [comment]
`fold(function (acc, v) { ... })` is fine. Just give me `forEach(function (v) { ... })` :D
